src/herder/Upgrades.cpp

```diff
@@ -530,7 +531,7 @@ enum class UpdateOfferResult
     Unchanged,
     Adjusted,
     AdjustedToZero,
-    Erased
+    Erase
```

Comment:
My reasoning for this was that `UpdateOffer` actually adjusts the offers, but it no longer does the erase. It could be misleading to see `Erased` even when the offerEntry still exists, so I changed the name.